paseos/tests/actor_builder_test.py

```diff
@@ -2,6 +2,8 @@
 import numpy as np
 import pykep as pk
 import sys
+import plotly.graph_objects as go
```

Comment:
would avoid adding plotly just for this to the dependencies

Reply:
Have removed it